app/(technician)/visit/[id]/layout.tsx

```diff
@@ -2,7 +2,7 @@ import BackButton from "@/components/navigation/BackButton";
 import NavigationBar from "@/components/navigation/NavigationBar";
 import { ReactNode } from "react";
 
-export default function Layout({children}: {children: ReactNode}) {
+export default function Layout({ children }: { children: ReactNode }) {
```

Comment:
I like the added whitespace.

Question: can we add a linter to enforce (or do the whitespace for us)?

Reply:
Yep, we can add a formatter for the codebase. That'd be a separate issue